server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/test/java/org/apache/james/webadmin/integration/UnauthorizedEndpointsTest.java

```diff
@@ -44,19 +45,16 @@
 import org.apache.james.webadmin.vault.routes.DeletedMessagesVaultRoutes;
 import org.eclipse.jetty.http.HttpStatus;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.extension.RegisterExtension;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
 import io.restassured.RestAssured;
 
-class UnauthorizedEndpointsTest {
-    @RegisterExtension
-    static CassandraJmapExtension cassandraJmapExtension = new CassandraJmapExtension(CassandraJmapExtension.JamesLifeCyclePolicy.COMMON_TO_ALL_TESTS);
+public abstract class UnauthorizedEndpointsTest {
```

Comment:
Also, we should annotate this class with @category(BasicFeature.class) as this is quite critical (and master current behaviour)